src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
 // write your code here
+const populationData = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const array = Array.from(populationData).map(
+  (el) => Number(el.textContent.replace(/,/g, '')),
+  // eslint-disable-next-line function-paren-newline
+);
```

Comment:
Consider renaming the variable 'array' to something more descriptive, like 'populationNumbers', to enhance code readability.